### PR TITLE
[i2c,doc] Update DV page simulation results link

### PR DESCRIPTION
**Pull request by crteja**
Update DV results of i2c DV page to
https://reports.opentitan.org/hw/ip/i2c/dv/latest/report.html

### CODE REVIEW

**Review by marnovandermaas — APPROVED** [review]
LGTM, I have seen this issue before and thought I had fixed it throughout the repo, but apparently I missed this one.

**Comment by crteja** [comment]
> Would you mind doing the same correction in this PR for the templates, OTBN, SRAM control, Earlgrey and autogen?

Sure, will do.

File: hw/top_earlgrey/ip_autogen/alert_handler/doc/dv/index.md [hunk 1]
```diff
@@ -15,7 +15,7 @@ title: "ALERT_HANDLER DV document"
 ## Current status
 * [Design & verification stage]({{< relref "hw" >}})
   * [HW development stages]({{< relref "doc/project/development_stages" >}})
-* [Simulation results](https://reports.opentitan.org/hw/ip/alert_handler/dv/latest/results.html)
+* [Simulation results](https://reports.opentitan.org/hw/ip/alert_handler/dv/latest/report.html)
```
Comment (marnovandermaas) [inline hw/top_earlgrey/ip_autogen/alert_handler/doc/dv/index.md:18]: This should be: https://reports.opentitan.org/hw/top_earlgrey/ip_autogen/alert_handler/dv/latest/report.html

File: hw/ip_templates/alert_handler/doc/dv/index.md [hunk 2]
```diff
@@ -15,7 +15,7 @@ title: "ALERT_HANDLER DV document"
 ## Current status
 * [Design & verification stage]({{< relref "hw" >}})
   * [HW development stages]({{< relref "doc/project/development_stages" >}})
-* [Simulation results](https://reports.opentitan.org/hw/ip/alert_handler/dv/latest/results.html)
+* [Simulation results](https://reports.opentitan.org/hw/ip/alert_handler/dv/latest/report.html)
```
Comment (marnovandermaas) [inline hw/ip_templates/alert_handler/doc/dv/index.md:18]: This should be: https://reports.opentitan.org/hw/top_earlgrey/ip_autogen/alert_handler/dv/latest/report.html

**Review by marnovandermaas — CHANGES_REQUESTED** [review]
Two links need fixing.

File: hw/top_earlgrey/ip_autogen/alert_handler/doc/dv/index.md (hunk 1 above)
Comment (crteja) [reply hw/top_earlgrey/ip_autogen/alert_handler/doc/dv/index.md]: Updated in [eb9d914](https://github.com/lowRISC/opentitan/compare/174738e3f56025c3f28bd6554b1c105c5c6513c3..eb9d9143f653b9d439d62694caf249d24cac12c5)

File: hw/ip_templates/alert_handler/doc/dv/index.md (hunk 2 above)
Comment (crteja) [reply hw/ip_templates/alert_handler/doc/dv/index.md]: Updated in [eb9d914](https://github.com/lowRISC/opentitan/compare/174738e3f56025c3f28bd6554b1c105c5c6513c3..eb9d9143f653b9d439d62694caf249d24cac12c5)

**Comment by mundaym** [comment]
@crteja Thanks for making this change! Unfortunately this needs to be rebased again before it can be submitted.

**Comment by crteja** [comment]
Thanks @mundaym. Rebased changes https://github.com/lowRISC/opentitan/pull/17439/commits/0be9f043bde87dd05a458e54029d75e1a8b9ab49

**Pull request merged**